tools/mock-service-host/src/mid/coordinator.ts

```diff
@@ -416,8 +416,10 @@ export class Coordinator {
     private setStatusToSuccess(ret: any) {
         // set status to succeed to stop polling
         if (ret) {
-            if (_.has(ret, 'status')) {
-                ret.status = 'Succeeded'
+            try {
+                ret['status'] = 'Succeeded'
+            } catch (err) {
```

Comment:
the `err` is not used in the exception handling block, so we don't need to declare it?

Reply:
It cannot be omitted because of lint error.